Backoffice/NavigationPanel/Strategies/AbstractNavigationPanelStrategy.php

```diff
@@ -23,17 +24,25 @@
     protected $role = null;
 
     /**
-     * @param string $name
-     * @param string $role
-     * @param int    $weight
-     * @param string $parent
+     * @param string                   $name
+     * @param string                   $role
+     * @param int                      $weight
+     * @param string                   $parent
+     * @param array                    $datatableParameter
+     * @param TranslatorInterface|null $translator
      */
-    public function __construct($name, $role, $weight, $parent)
+    public function __construct($name, $role, $weight, $parent, $datatableParameter, $translator)
     {
         $this->name = $name;
         $this->role = $role;
         $this->weight = $weight;
         $this->parent = $parent;
+
+        if (!is_null($translator)) {
```

Comment:
My purpose was not on the validity of the check, but rather on the method signature, you should have `$translator = null` this way, you are not entitled to put null in the constructor when you call it.

My second point is to modify the check in `$translator instanceof TranslatorInterface`